### ISSUE TITLE
PDF Form is Likely not Accessible.mp4

### ISSUE THREAD

**Issue by jill-iterators** [CityOfBoston/Iterators#147]
Screengrab/videos (insert file names here)
2020 08 24, PDF Likely Not Accessible.mp4

Link
https://d8-stg.boston.gov/departments/emergency-medical-services/how-apply-ems-special-event-permit

Node # (if known)

Type of User:  Anonymous

Environment
(Device > OS > Browser)
Computer > MacOS 10.15.5 > Chrome 84

Trusted Tester Test
n/a

Brief Description
While Accessibility testing, PDF populated and likely not accessible. This should be checked as a general form that is on the site.

Steps to reproduce
1)	00:00, Go to site, with link provided above
2)	00:07, Click on the link titled Emergency Medical Services Permit Application.
3)	00:12, PDF form appears, form must be printed and returned, not editable online. 
a)	This form likely will fail the Accessibility testing and should be checked. PDF forms require a different type of testing that is currently not in scope. 

Desired Result
•	Able to use forms found on the website and be able to use them as a person with some form of accessibility issue.

Actual Result
•	Unable to use forms found on the website and be able to use them as a person with some form of accessibility issue.

**Comment by subaha-cob** [comment]
@jill-iterators I am going to need more information and a video link that I can watch. Thanks.

**Comment by mmcgowanBOS** [comment]
HI @jill-iterators 
I had some time to discuss this issue with our team and this is a pretty large site wide issue that I think we'll have to tackle in a separate effort. For now we may hold off on immediately addressing, but can circle back once we have all other issues addressed and ready for final review.

**Comment by reillyzlab** [comment]
We will be closing this issue. 

While we'd like to do this for every form on the website there is not standardized 'policy' from the City that all forms need to be editable. When we find them we encourage departments to create them as editable pdfs.

However, most departments don't have software licenses to create these editable forms, but they do have scanners, so they'll scan a doc and just upload. 

For this form specifically, we are reviewing all BPHC pdf forms as part of another website migration project.
